package.json

```diff
@@ -163,6 +164,7 @@
     "css-loader": "0.28.4",
     "cypress": "^3.1.5",
     "dotenv": "4.0.0",
+    "emotion-theming": "^10.0.10",
```

Comment:
Wow. I'm surprised an Emotion 10 package didn't blow up Fancy! (which is on Emotion 9)

Reply:
I was worried about that too and for a moment thought it could be a blocker... but seems fine 🤞